README.md

```diff
@@ -97,6 +97,14 @@ using different configurations. Check the `./getting-started` directory for more
 - `kubectl get deployment -n polaris` - To check the status of the deployment.
 - `kubectl describe deployment polaris-deployment -n polaris` - To troubleshoot if things aren't working as expected.
 
+#### Configuring Polaris
+
+Polaris Servers can be configured using a variety of ways.
+Please see the [Configuration Guide](https://polaris.apache.org/in-dev/unreleased/configuration/)
+for more information.
+
+Default configuration values can be found in `quarkus/defaults/src/main/resources/application.properties`.
```

Comment:
Not a blocker to this PR, but I'm still concerned as this file contains both build-time and runtime configs. I'm not sure if there is a better way to reduce the confusion from the users, other than documenting them well. Open for suggestions.

Reply:
Would it help if we had a different section for build-time properties in `application.properties` with a comment explaining how they work?

Reply:
yeah, that definitely helps.

Reply:
#970